content/kubermatic/master/_index.en.md

```diff
@@ -9,30 +9,73 @@ date = 2019-04-27T16:06:34+02:00
 
 ## What is Kubermatic Kubernetes Platform (KKP)?
 
-KKP is one enterprise Kubernetes management platform for any infrastructure. KKP automates thousands of Kubernetes clusters across multi-cloud, on-prem and edge with unparalleled density and resilience. Deploy, manage and run multiple Kubernetes clusters with our production-proven platform on your preferred infrastructure.
+Kubermatic Kubernetes Platform (KKP) is a Kubernetes management platform that helps address the operational and security challenges of enterprise customers seeking to run Kubernetes at scale. KKP automates deployment and operations of hundreds or thousands of Kubernetes clusters across hybrid-cloud, multi-cloud and edge environments while enabling DevOps teams with a self-service developer and operations portal.
```

Comment:
> ... while enabling DevOps teams with a self-service developer and operations portal.

What does the developer portal mean in this sense? I guess you want to highlight that this is a portal to be used by both devs and ops folks, but IMO, this reads like a portal that you would use for development.

Reply:
It means a self-service portal for DevOps teams. This is used in most of our communications-do you have an alternate suggestion?

Reply:
Let's leave it as it is.